src/js/rx/containers/History.jsx

```diff
@@ -146,10 +146,13 @@ class History extends React.Component {
       content = (
         <div>
           <p className="rx-tab-explainer">Your VA prescription refill history.</p>
-          <SortMenu
-              onChange={this.handleSort}
-              options={fields}
-              selected={currentSort.value}/>
+          <div className="show-for-small-only">
```

Comment:
Okay, works for me. I didn't know about it before, but it can be useful for some one-off cases like this.